client/state/sites/selectors/get-env-stats-feature-supports.ts

```diff
@@ -51,6 +51,8 @@ function getEnvStatsFeatureSupportChecks( state: object, siteId: number | null )
 			// UTM stats are only available for Jetpack sites for now.
 			isSiteJetpackNotAtomic &&
 			version_greater_than_or_equal( statsAdminVersion, '0.17.0-alpha', isOdysseyStats ),
+		supportsDevicesStats:
+			config.isEnabled( 'stats/devices' ) && isSiteJetpackNotAtomic && isOdysseyStats,
```

Comment:
We need to remove the `isOdysseyStats` check to see the module on Calypso.

```suggestion
			config.isEnabled( 'stats/devices' ) && isSiteJetpackNotAtomic,
```